src/conda/env_finite_amplitude_wave_diag.yml

```diff
@@ -0,0 +1,18 @@
+name: _MDTF_finite_amplitude_wave_diag
+channels:
+- conda-forge
+dependencies:
+- python=3.12
+- numpy=1.26.4
+- scipy=1.14.0
+- netCDF4=1.6.5
+- xarray=2024.1.1
+- cartopy=0.22.0
+- matplotlib=3.8.2
+- pytest==7.4.0
+- dask
+- gridfill
+- bottleneck
+- pip=23.3.1
+- pip:
+    - git+https://github.com/csyhuang/hn2016_falwa.git
```

Comment:
@wrongkindofdoctor Hi Jess, we figured out a solution to upgrade `falwa` to python 3.12 by migrating the F2PY build to meson. We tested on both Mac and Linux machine that the new installation procedures work. Do you have a chance to test if you can get our package installed via
```
- pip:
    - git+https://github.com/csyhuang/hn2016_falwa.git
```
Thanks and we look forward to your update!

Clare

Reply:
@csyhuang Excellent! I'm glad you were able to find a replacement for F2PY. Yes, please push the update to PyPI and reference the package instead of the git repo in the environment file.

Reply:
@wrongkindofdoctor I've just pushed an update to the PR branch and changed the line to `- falwa==2.0.0`. Thanks Jess!

Reply:
@csyhuang Thanks! Please move your environment file and runtime config file from  /src to your POD directory; I will add falwa and the dependencies back to the python3 base env files and verify that they build in the in the CI.

Reply:
@wrongkindofdoctor Sure! I've moved the files to `diagnostics/finite_amplitude_wave_diag`. Please keep us updated after you tested it! Thanks Jess!

Reply:
@csyhuang The environments have been tested and merged into the python3_base env files.

Reply:
@wrongkindofdoctor Glad that it works! Thanks Jess! We'll update you after finishing the rest of our POD!